docs/testcases/TC06-009-fail.html

```diff
@@ -79,7 +79,7 @@ <h4 class="bg-light border-left mb-0 p-2">Table of Contents</h4>
   						<h1 class="card-title" id="TCtitle">Image - with aria-hidden="true"</h1>
   							<h2 class="border-bottom" id="TCsum">Test Case Summary</h2>
   								<h3 id="TCid">Test Case ID</h3>
-  									<p>TC06-008-fail</p>
+  									<p>TC06-009-fail</p>
   								<h3 id="TCdesc">Test Case Description</h3>
   									<p>Detect existence of attributes that would contribute to the accessible name and accessible description computation and calculate the text alternative for the image. Then determine whether the text alternative output for the image provides an equivalent description of the image.</p>
   									<p>The code sample provides an equivalent description of the image in an ALT attribute. Hwoever, because it includes <code>aria-hidden="true"</code>, Assistive Technologies should ignore the presence of the image. A successful test should identify a FAIL against Baseline 6. Images because the meaningful image DOES NOT provide an equivalent text description.</p>
```

Comment:
Two issues for TC06-009-fail.html:

- Typo of "Hwoever" under "Test Case Description" 

- Minor issue: The code `<p>` value does not match the actual `<p>` value in the embedded test file. The difference is minor but different.

Reply:
Fixed